### PR TITLE
Upgraded Spring dependency version to 2.1.15.RELEASE as part of GCI 1269

### PR DESCRIPTION
**Pull request by YiannisPapastergiou**
Amendments made to upgrade spring boot to 2.1.15.RELEASE.

Jira ticket - GCI-1269

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -36,8 +35,8 @@
         <structured-logging.version>1.4.0-rc2</structured-logging.version>
 
         <!-- Maven and Surefire plugins -->
-        <maven-compiler-plugin.version>3.1</maven-compiler-plugin.version>
-        <maven-surefire-plugin.version>2.21.0</maven-surefire-plugin.version>
+        <maven-compiler-plugin.version>3.8.1</maven-compiler-plugin.version>
+		<maven-surefire-plugin.version>2.22.2</maven-surefire-plugin.version>
```
Comment (hwilliams1) [inline pom.xml:39]: Can this be formatted to bring it in line with the rest of the entries please?

File: pom.xml [hunk 2]
```diff
@@ -52,7 +51,22 @@
         <artifactoryResolveReleaseRepo>virtual-release</artifactoryResolveReleaseRepo>
 
         <environment-reader-library.version>1.2.0-rc1</environment-reader-library.version>
+        
+		<spring-boot-dependencies.version>2.1.15.RELEASE</spring-boot-dependencies.version>
+		<spring-boot-maven-plugin.version>2.1.15.RELEASE</spring-boot-maven-plugin.version>
```
Comment (hwilliams1) [inline pom.xml:56]: Formatting of these 2 lines - indented too far. Can this be fixed please?

File: pom.xml [hunk 3]
```diff
@@ -52,7 +51,22 @@
...
+		<dependencies>
+			<dependency>
+				<groupId>org.springframework.boot</groupId>
+				<artifactId>spring-boot-dependencies</artifactId>
+				<version>${spring-boot-dependencies.version}</version>
+				<type>pom</type>
+				<scope>import</scope>
+			</dependency>
+		</dependencies>
+	</dependencyManagement>
```
Comment (hwilliams1) [inline pom.xml:69]: The indentation should be 4 characters per tab, this looks like it is something like 10 characters? Please can this be re-formatted to be 4 spaces per tab.

Comment (YiannisPapastergiou) [reply pom.xml]: This should be correct now

**Pull request merged**